core/coreobjects/include/coreobjects/property_object_impl.h

```diff
@@ -766,6 +767,20 @@ ErrCode GenericPropertyObjectImpl<PropObjInterface, Interfaces...>::setPropertyV
             validatePropertyWrite(prop, valuePtr);
             coerceMinMax(prop, valuePtr);
 
+            const auto ct = prop.getValueType();
+            if (ct == ctList)
+            {
+                ListPtr<IBaseObject> listValue;
+                valuePtr.asPtr<ICloneable<IList>>()->clone(&listValue);
```

Comment:
I like to get in the habit of calling checkErrorInfo on pure interface calls that return ErrCode.